GoogleCloudPlatform/datacatalog-connectors-bi
# Create Tags for Custom Property Values

**- What I did**
Added the required code to create Tags for Custom Property Values.

**- How I did it**
Added a mechanism that creates a new Tag Template for each `choiceValue` found in the Custom Property Definitions. Then, such Tag Templates are used to store all metadata from the Custom Property Values. Each Value is stored as a Tag.

Please notice the `qlik/prepare/dynamic_properties_helper.py` file might be part of the commons package. Maybe we can move it as a follow-up change.

**- How to verify it**
Run the unit tests or, preferably, the Qlik Sense connector against a Qlik site that contains Streams and Apps with Custom Properties. Then, check the Tags created for such assets.

**- Description for the changelog**
Added support for creating Tags for Custom Property Values
